docs/onderwerpen/data.md

```diff
@@ -20,11 +20,11 @@ Net als organisaties mogen algoritmes niet zomaar gegevens verzamelen en gebruik
 Andere belangrijke regels gaan over privacy. Zo mag je algoritme alleen de minimale persoonsgegevens gebruiken die nodig zijn om het doel te bereiken. Technieken om dit te doen zijn:
 
 * Anonimiseren: data zoveel mogelijk anoniem maken
-* [Pseudonimiseren](https://www.autoriteitpersoonsgegevens.nl/themas/beveiliging/beveiliging-van-persoonsgegevens/gegevens-pseudonimiseren): data moeilijker herleidbaar maken naar personen
+* [Pseudonimiseren](https://www.autoriteitpersoonsgegevens.nl/themas/beveiliging/beveiliging-van-persoonsgegevens/gegevens-pseudonimiseren): data moeilijker herleidbaar maken (veelal in geval van persoonsgegevens)
 * Aggregeren: data zoveel mogelijk combineren of samenvoegen tot 1 waarde, zoals een totaal of gemiddelde 
 
 ### Goede datakwaliteit
-Hoe slechter de datakwaliteit, hoe onbetrouwbaarder de uitkomsten van je algoritme. 
+Voor algoritmes die getrained worden geldt dat een slechte datakwaliteit ook leidt tot algoritmes met een gebrekkige validiteit.
```

Comment:
Goede datakwaliteit is in het geval van het trainen of ontwikkelen van een algoritme belangrijk zodat het algoritme een zo hoog mogelijke validiteit krijgt. Bij het gebruik van een zelflerend algoritme kan slechte datakwaliteit de validiteit van het algoritme verlagen. Bij rekenregels met een hoge validiteit wordt slechte datakwaliteit juist aangetoond, maar heeft dat geen invloed op de algoritme zelf. Denk bij dat laatste aan fraude opsporende algoritmes.